signer/remote_signer.go

```diff
@@ -103,7 +116,7 @@ func (rs *ReconnRemoteSigner) loop() {
 				conn = nil
 				rs.Logger.Error("Secret Conn", "err", err)
 				rs.Logger.Info("Retrying", "sleep (s)", 3, "address", rs.address)
-				time.Sleep(time.Second * 3)
+				time.Sleep(time.Second * 1)
```

Comment:
A ticker may be more idiomatic. It could remove the need to call `Sleep` twice.

Fyi, `time.Sleep` is rare for idiomatic Go - usually indicates a code smell or hack.

Reply:
Yes, been meaning to clean this one up too. It's an old copy paste from the tendermint code base.

Reply:
done